src/include/reconcile.h

```diff
@@ -227,6 +227,14 @@ struct __wt_reconcile {
     size_t supd_allocated;
     size_t supd_memsize; /* Size of saved update structures */
 
+    /*
+     * History store update to be deleted list. While reviewing updates for each page, we save
+     * WT_HS_DELETE_UPD lists here, and then delete them after we have built the disk image.
+     */
+    WT_DELETE_HS_UPD *delete_hs_upd; /* Updates to delete from history store */
+    uint32_t delete_hs_upd_next;
```

Comment:
Should we call it `delete_hs_upd_size` or `delete_hs_upd_count`?

Reply:
The name is consistent to other similar use cases. I intend to keep the same pattern.